buildtest/cli/__init__.py

```diff
@@ -1020,6 +1020,10 @@ def config_menu(subparsers, parent_parser):
     )
 
     subparsers_config.add_parser("systems", help="List all available systems")
+    #buildtest config profile remove
+    subparsers_profile_list.add_argument(
+        "-y", "--yaml", action="store_true", help="List Profile details in YAML Format"
+    )
```

Comment:
why are you declaring this. You already have this declared above in lines 1006. You need to declare a subparser `buildtest config profiles remove` and this code is not doing that. Please look at original issue and make sure you get this working correctly.